### PR TITLE
[DOCS-6229] Add link to dd-sdk-android version for logs collection

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

Adds a link to the dd-sdk-android version.
[DOCS-6229](https://datadoghq.atlassian.net/browse/DOCS-6229)

### Merge instructions

- [ ] Please merge after reviewing

[DOCS-6229]: https://datadoghq.atlassian.net/browse/DOCS-6229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by buraizu — APPROVED** [review]
These changes LGTM, but I think you might need to update the branch to get the build preview check to pass

File: content/en/logs/log_collection/android.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 
 ## Setup
 
-1. Add the Gradle dependency by declaring the library as a dependency in the module-level `build.gradle` file:
+1. Add the Gradle dependency by declaring the library as a dependency in the module-level `build.gradle` file. Make sure to replace `x.x.x` in the following example with the latest version of [dd-sdk-android][2].
```
Comment (0xnm) [inline content/en/logs/log_collection/android.md:26]:
```suggestion
1. Add the Gradle dependency by declaring the library as a dependency in the module-level `build.gradle` file. Make sure to replace `x.x.x` in the following example with the latest version of [dd-sdk-android-logs][2].
```

**Review by 0xnm — CHANGES_REQUESTED** [review]
I've added a suggestion related to the reference to `dd-sdk-android-logs` instead of `dd-sdk-android`. `dd-sdk-android` as an artifact doesn't exist anymore, we have multiple modules now (per feature) which all share the same version.

**Review by 0xnm — APPROVED** [review]

**Comment by maycmlee** [comment]
> I've added a suggestion related to the reference to `dd-sdk-android-logs` instead of `dd-sdk-android`. `dd-sdk-android` as an artifact doesn't exist anymore, we have multiple modules now (per feature) which all share the same version.

Thanks @0xnm for the info! I committed the change.

**Pull request merged**